.github/workflows/cli-release-linux.yaml

```diff
@@ -1,33 +1,41 @@
 name: cli-release-build-native-linux
 
 on:
+  workflow_dispatch: {}
   release:
-    types: [created]
-  workflow_dispatch:
-    inputs:
-      tag_name:
-        description: 'Release Version (e.g. 2.0.0.Final)'
-        required: true
+    types: [released]
 
 jobs:
   build:
+    if: github.repository_owner == 'Apicurio'
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
```

Comment:
Existing workflow always checks out master which is not correct. The latest release was done from 2.0.x branch and therefore the build should have been done from the respective branch. But the CLI tools uploaded to the release asset was built from the master branch.

Reply:
when you trigger the workflow it asked for the branch to run the workflow on, I think it did checkout the code from the release branch